### PR TITLE
Update the workaround in the onExporting option

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 1]
```diff
@@ -27,137 +27,216 @@ The name of the file where grid data is about to be exported.
...
+                workbook.xlsx
+                    .writeBuffer()
+                    .then(function(buffer) {
+                    saveAs(
+                        new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx");
+                    })
+                    .then(() => {
+                        e.component.columnOption("ID", "visible", false);
+                        e.component.endUpdate();
+                    });
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]: Alignments are messed up.

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 2]
```diff
@@ -27,137 +27,216 @@ The name of the file where grid data is about to be exported.
...
+                    workbook.xlsx
+                        .writeBuffer()
+                        .then(function(buffer) {
+                        saveAs(
+                            new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx");
+                        })
+                        .then(() => {
+                            e.component.columnOption("ID", "visible", false);
+                            e.component.endUpdate();
+                        });
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]: The same problem with alignments.

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 3]
```diff
@@ -27,137 +27,216 @@ The name of the file where grid data is about to be exported.
...
+                workbook.xlsx
+                    .writeBuffer()
+                    .then(function(buffer) {
+                    saveAs(
+                        new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx");
+                    })
+                    .then(() => {
+                        e.component.columnOption("ID", "visible", false);
+                        e.component.endUpdate();
+                    });
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]: And here.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 4]
```diff
@@ -27,137 +27,216 @@ The name of the file where grid data is about to be exported.
...
-            e.component.columnOption("ID", "visible", false);
-            e.component.endUpdate();
+            const workbook = new ExcelJS.Workbook();
+            const worksheet = workbook.addWorksheet("Employees");
+
+            exportDataGrid({
+                component: e.component,
+                worksheet: worksheet
+                }).then(function() {
+                workbook.xlsx
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]: In this code example need to fix the indentations with formatting. 
Check it also in all approaches, pls

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 5]
```diff
@@ -27,137 +27,216 @@ The name of the file where grid data is about to be exported.
...
-
         onExporting(e) {
             e.component.beginUpdate();
-            e.component.columnOption('ID', 'visible', true);
-        }
-
-        onExported(e) {
-            e.component.columnOption('ID', 'visible', false);
-            e.component.endUpdate();
+            e.component.columnOption("ID", "visible", true);
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]: Should we not to use the single quotes for js code in snippets?

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 6]
```diff
@@ -27,133 +27,205 @@ The name of the file where grid data is about to be exported.
...
-            @exporting="onExporting"
-            @exported="onExported"
-        >
-            <DxColumn
-                data-field="ID"
-                :visible="false"
-            />
+            @exporting="onExporting">
+            <DxExport :enabled="true" />
+            <DxColumn data-field='ID' :visible="false" />
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]: Attributes should use double quotes.

```suggestion
            <DxColumn data-field="ID" :visible="false" />
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 7]
```diff
@@ -27,133 +27,205 @@ The name of the file where grid data is about to be exported.
...
+##### Angular   
 
-    <!--TypeScript-->
-    import { DxDataGridModule } from "devextreme-angular";
-    // ...
+    <!-- tab: app.component.html -->
+    <dx-data-grid ...
+        (onExporting)="onExporting($event)">
+        <dxo-export [enabled]="true"></dxo-export>
+        <dxi-column dataField='ID' [visible]="false"></dxi-column>
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]:
```suggestion
        <dxi-column dataField="ID" [visible]="false"></dxi-column>
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md [hunk 8]
```diff
@@ -27,133 +27,205 @@ The name of the file where grid data is about to be exported.
...
         render() {
             return (
-                <DataGrid ... 
-                    onExporting={this.onExporting}
-                    onExported={this.onExported}>
-                    <Column dataField="ID" visible={false} />    
+                <DataGrid ...
+                    onExporting={this.onExporting}>
+                    <Export enabled={true} />
+                    <Column dataField='ID' visible={false} />
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onExporting.md]:
```suggestion
                    <Column dataField="ID" visible={false} />
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**